realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -697,6 +703,8 @@ void remove(Class<? extends RealmObject> clazz, long objectIndex) {
      * will not be reflected in the Realm copy. This is a deep copy, so all referenced objects will be copied. Objects
      * already in this Realm will be ignored.
      *
+     * Please note, all unset fields will be set to their default value if not provided.
```

Comment:
Can you talk about unset fields in objects? They will implicitly always have a value.